### PR TITLE
Update to Readme.md

### PR DESCRIPTION
**Pull request by Kuassim**
Made some changes and introduced Free-Tier, Always-Free

### CODE REVIEW

**Comment by Kuassim** [comment]
Removed "build" from the 3rd use case.
I have observed further issues that will be addressed in my next review.

File: README.md [hunk 1]
```diff
@@ -1,61 +1,67 @@
...
-- **DNS:** A DNS zone to create the application URL (for example https://myapp.domain.com). If not provided during the stack configuration, the application will be available through the load balancer's public IP. You can then configure your third-party DNS provider to point to this IP address.
-- **HTTPS certificate:** is needed for the load balancer. If no certificate is provided, HTTP will be used against the IP address.
-- **Database:** an existing ADB can be used with the stack. An option is also provided to make the stack create one if needed.
+- **DevOps project (optional):** A Java application in an OCI DevOps project (can be a mirror of an existing GitHub repo). This isn't required if the application is provided as a container image. Note that the repo in OCI DevOps can be a mirror of a GutHub repo.
```
Comment (jeandelavarene) [inline README.md:37]: Can you fix the repetition here?

File: README.md [hunk 2]
```diff
@@ -1,61 +1,67 @@
...
+- **Database:** an existing Autonomous Database  - Shared Infrastructure (ADB-S) can be used with the stack. The stack may create a new one, if specified during the Stack configuration. 
+- **Vault (optional):** A new user in IAM (<application_name>-user) is created and his token used for connectng to the DevOps repo, is stored in the vault. When the stack is destroyed this user is removed. A Vault is necessary to avoid the limit on the number of tokens the current user have; we do not recommend using the current user however, the Vault isn't required if the application is provided as a container image.
```
Comment (jeandelavarene) [inline README.md:39]: We never use the current user.

**Comment by Kuassim** [comment]
Addressed all comments.

**Review by jeandelavarene — APPROVED** [review]

**Pull request merged**